tests/e2e/tests/releng_utils.py

```diff
@@ -0,0 +1,106 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import requests
+
+
+class FirefoxLocale:
+
+    def __init__(self, locale, version_info):
+        self.lang = locale
+        self.versions = self.parse_versions(version_info)
+
+    def parse_versions(self, version_info):
+        """Parses a dict and returns a list of Firefox versions,
+        ignoring Auroa and esr builds.
+
+        :arg version_info: {string:string} A dictionary with k,v of version and
+        os.
+
+        :returns: [string] A list of Firefox versions
+        """
+        versions = []
+        for version, os in version_info.items():
+            # remove all aurora builds and esr builds
+            if 'a' not in version and 'esr' not in version:
+                versions.append(version)
+        return versions
+
+    def __repr__(self):
+        return self.locale
+
+
+_base_url = 'https://product-details.mozilla.org/1.0/%s'
+_firefox_primary_builds_uri = 'firefox_primary_builds.json'
+_firefox_versions_uri = 'firefox_versions.json'
+# mappings adapted off of https://github.com/mozilla-releng/ship-it/blob/master/kickoff/config.py
+releng_to_bouncer_alias_dict = {
+    'FIREFOX_AURORA': 'firefox-aurora-latest',
+    'FIREFOX_ESR_NEXT': 'firefox-esr-latest',
```

Comment:
`ESR_NEXT` is not always set to some version. We have this variable set only for 2 release cycles (12 weeks), when we have 2 supported ESR versions. You may end up with test failures when we stop serving ESR45 and ESR52 will be the only ESR version. At that point `FIREFOX_ESR` will be the version you would need to test against.